internal/controllers/s3userclaim/provisioner.go

```diff
@@ -345,6 +347,38 @@ func (r *Reconciler) updateClusterQuotaStatus(ctx context.Context, addCurrentQuo
 
 	return subreconciler.ContinueReconciling()
 }
+
+func (r *Reconciler) assignNamespaceQuotatoResourceStatus(statusNamespaces openshiftquota.ResourceQuotasStatusByNamespace) openshiftquota.ResourceQuotasStatusByNamespace {
+	if r.namespaceUsedQuota == nil {
+		r.logger.Error(consts.ErrNamespaceQuotaNotDefined,
```

Comment:
If this is a real error why isn't the code returning?
I think we should use `warn` log level here.

Reply:
Yes, you're right. I intended to do that but [logr](https://pkg.go.dev/github.com/go-logr/logr) package doesn't have the `warn` level. 
As I find out, we have two options:
1. `r.logger.Info` with a warning message containing the warning term
2. `r.logger.Error(nil, warning message)`

Do you have any suggestion?

Reply:
Absolutely info 😃.

Reply:
Done.